assets/css/all.sass

```diff
@@ -53,3 +53,7 @@ a
 
 \:target
   background: #ddd
+
+.question
+  font-size: 1.25em
```

Comment:
nit: perhaps a bit smaller (maybe same size as the ### subtitle is in this PR)

Reply:
changed to 1.2